docarray/array/storage/redis/seqlike.py

```diff
@@ -27,7 +27,7 @@ def __len__(self):
         :return: the length of this :class:`DocumentArrayRedis` object
         """
         try:
-            return len(self._offset2ids)
```

Comment:
u can try to see if `offset2ids` exists and then default to this, it may be faster